Libplanet/Blocks/MultipleTxsOfSameSignerException.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+
+namespace Libplanet.Blocks
+{
+    /// <summary>
+    /// The exception that is thrown when the namespace of
```

Comment:
What the term “namespace” does mean here?

Reply:
My mistake. Fixed.